verticapy/core/vdataframe/_multiprocessing.py

```diff
@@ -27,6 +27,9 @@
 def aggregate_parallel_block(
```

Comment:
I couldn't tell whether these were intended to be internal or not. If they are going to be public, might need some help w/ the descriptions for these two functions